tests/unit/type/list.tcl

```diff
@@ -1913,8 +1912,9 @@ foreach {type large} [array get largevalue] {
 
         $rd1 brpoplpush a b 0
         $rd1 brpoplpush a b 0
+        wait_for_blocked_clients_count 1
         $rd2 brpoplpush b c 0
-        after 1000
```

Comment:
i see `$pop: with zero timeout should block indefinitel` also have a `after 1000`